src/tests/ftest/util/dmg_utils.py

```diff
@@ -321,6 +321,43 @@ def storage_query_list_pools(self, uuid=None, rank=None, verbose=False):
             ("storage", "query", "list-pools"), uuid=uuid, rank=rank,
             verbose=verbose)
 
+    def storage_identify_vmd(self, uuid=None, verbose=False):
```

Comment:
Isn't uuid required?

Reply:
It is required. If someone doesn't provide, the command will fail. Using similar template as other method. Anything wrong?

Reply:
If required, I don't think we should set a default of `None`:
```
def storage_identify_vmd(self, uuid, verbose=False):
```

Reply:
For example, this existing function:
```
def storage_set_faulty(self, uuid, force=True):
```

Reply:
Will do it.

Reply:
Done.